steps/mentor-calibre-lvs/configure.yml

```diff
@@ -4,6 +4,9 @@
 # Author : Christopher Torng
 # Date   : November 5, 2019
 #
+# [steveri 01/2021] Added lvs2, lvs_extra_spice etc. to support where e.g.
+# most of the chip uses stdview but one IP block (dragonphy in our case)
+# needs a different view b/c it uses LVT cells etc.
```

Comment:
So in the end I think the only change is the single line to add the parameter for "lvs_extra_spice_include".